### PR TITLE
Feat: Change background image to solid color

### PR DESCRIPTION
**Pull request by zunamidev**
Added setting to disable the background image and change it to a solid color which is currently not choosable.

### CODE REVIEW

**Comment by zunamidev** [comment]
#292

File: client/.env.sample [hunk 1]
```diff
@@ -1,2 +1,2 @@
 NODE_ENV=development
-VUE_APP_API_URL=http://localhost:3000
\ No newline at end of file
+VUE_APP_API_URL=http://localhost:3100
```
Comment (jayehernandez) [inline client/.env.sample:2]: Can we retain this to port 3000?

File: client/src/App.vue [hunk 2]
```diff
@@ -3,7 +3,8 @@
     div(v-if="hasError")
       ErrorPage
     div(v-else)
-      #background(:style="{ backgroundImage: `url('${imageUrl}')` }")
+      #background(:style="{ backgroundImage: `url('${imageUrl}')` }" v-if="styleOptions")
+      #background(:style="{ backgroundColor: `#000` }" v-else)
```
Comment (jayehernandez) [inline client/src/App.vue:7]: For `v-else` we can remove the style option and just add it directly in the CSS

File: client/src/views/Options.vue [hunk 3]
```diff
@@ -16,6 +16,14 @@ div(v-click-outside="closeOptionsMenu")
             @click="resetMessage"
           )
           span.checkmark
+      .mb1 Style
+        .choices
+          label.option Show background image
```
Comment (jayehernandez) [inline client/src/views/Options.vue:21]: I think the default for this is that the background is shown, unless a user clicks on this button and wants to remove it.

Can we change this to "Remove background image" instead?

**Review by jayehernandez — CHANGES_REQUESTED** [review]
Great PR so far! Thanks for finding the time to contribute to open source.

Just added a few comments to the files, also want to discuss the logic on how this will be saved. Right now from the PR, I think it will only be saving to vuex and not to Chrome Storage. This would mean that if a new tab is opened, the values they input before will not be saved.

We'll be needing to change the code for this so that it will be saving to Chrome as well, similar to how the preferred languages are saved. We can just add this to that call so that it will be a "saveSettings" method instead of the current save languages. [Here's](https://developer.chrome.com/apps/storage) a guide on this one.

I can help out with the local storage stuff too, if needed. Would you want to explore on that first? If not I can just add to this PR after the other comments are addressed. Let me know if you have any questions.

File: client/src/App.vue (hunk 2 above)
Comment (zunamidev) [reply client/src/App.vue:7]: You mean to add a class instead of the inline style?

File: client/.env.sample (hunk 1 above)
Comment (zunamidev) [reply client/.env.sample:2]: I changed it

File: client/src/App.vue (hunk 2 above)
Comment (jayehernandez) [reply client/src/App.vue:7]: Yep!

File: client/src/store/index.js [hunk 4]
```diff
@@ -108,6 +109,9 @@ export const mutations = {
   setLoading(state, loading) {
     state.loading = loading;
   },
+  setStyleOptions(state, styleOptions) {
```
Comment (jayehernandez) [inline client/src/store/index.js:112]: Apart from this, you'll also need to save the values in Chrome's storage, left a comment on your PR about it.

**Review by jayehernandez — CHANGES_REQUESTED** [review]

**Pull request closed without merging**